### PR TITLE
docs: add more way to install APISIX

### PR DESCRIPTION
**Pull request by spacewander**
Also prefer to install via package instead of source.
The latter one is not so friendly for beginner.

Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by moonming — APPROVED** [review]

File: docs/en/latest/how-to-build.md [hunk 1]
```diff
@@ -31,6 +31,20 @@ So before installation, please follow the different operating systems [install D
 
 You can install Apache APISIX in a variety of ways, including source code packages, Docker, and Luarocks.
 
+### Installation via RPM package (CentOS 7)
+
+```shell
```
Comment (juzhiyuan) [inline docs/en/latest/how-to-build.md:36]: I see we use `bash` and `shell`, do we need to use the same lang?

**Review by juzhiyuan — APPROVED** [review]

File: docs/en/latest/how-to-build.md (hunk 1 above)
Comment (spacewander) [reply docs/en/latest/how-to-build.md:36]: The original one used `shell`. Does it really matter to use only `bash` or `shell`? I don't want to waste time on a trivial thing.

Comment (juzhiyuan) [reply docs/en/latest/how-to-build.md:36]: No, just want to confirm.

Comment (spacewander) [reply docs/en/latest/how-to-build.md:36]: I see.

**Pull request merged**